src/dbg/debugger_commands.cpp

```diff
@@ -715,20 +728,76 @@ CMDRESULT cbDebugSetBPGoto(int argc, char* argv[])
     }
     char cmd[deflen];
     _snprintf(cmd, sizeof(cmd), "SetBreakpointCondition %s, 0", argv[1]);
-    if(!DbgCmdExecDirect(cmd))
+    if(!_dbg_dbgcmddirectexec(cmd))
         return STATUS_ERROR;
     _snprintf(cmd, sizeof(cmd), "SetBreakpointCommand %s, \"CIP=%s\"", argv[1], argv[2]);
-    if(!DbgCmdExecDirect(cmd))
+    if(!_dbg_dbgcmddirectexec(cmd))
         return STATUS_ERROR;
     _snprintf(cmd, sizeof(cmd), "SetBreakpointCommandCondition %s, 1", argv[1]);
-    if(!DbgCmdExecDirect(cmd))
+    if(!_dbg_dbgcmddirectexec(cmd))
         return STATUS_ERROR;
     _snprintf(cmd, sizeof(cmd), "SetBreakpointFastResume %s, 0", argv[1]);
-    if(!DbgCmdExecDirect(cmd))
+    if(!_dbg_dbgcmddirectexec(cmd))
         return STATUS_ERROR;
     return STATUS_CONTINUE;
 }
 
+CMDRESULT cbDebugSetBPHwKeep(int argc, char* argv[])
+{
+    // will this have an effect?

```

Comment:
Yep, you will get an access violation if the user called with no arguments and you try to access `argv[1]`.